README.rst

```diff
@@ -8,14 +8,14 @@ Cookiecutter
 .. image:: https://img.shields.io/pypi/pyversions/cookiecutter.svg
         :target: https://pypi.python.org/pypi/cookiecutter
 
-.. image:: https://travis-ci.org/audreyr/cookiecutter.svg?branch=master
-        :target: https://travis-ci.org/audreyr/cookiecutter
+.. image:: https://travis-ci.org/cookiecutter/cookiecutter.svg?branch=master
+        :target: https://travis-ci.org/cookiecutter/cookiecutter
```

Comment:
The builds linked to PRs point to `travis-ci.com` not `.org` 🤔

Reply:
We have travis.org and travis.com enabled, so I got confused. Either way, Travis-ci recommends [1][2] migrating to travis-ci.com, so let's do it.

[1] https://blog.travis-ci.com/2018-05-02-open-source-projects-on-travis-ci-com-with-github-apps
[2] https://docs.travis-ci.com/user/migrate/open-source-on-travis-ci-com/